scripts/ci.ps1

```diff
@@ -155,4 +155,4 @@ Enable-PSRemoting
 Set-WSManQuickConfig -Force
 Set-WinRMHostConfiguration
 Test-WinRMConfiguration @User | Out-Null
-Write-Output "::set-env name=BOLT_WINRM_PASSWORD::$pass"
\ No newline at end of file
+$ENV:GITHUB_ENV=@{ BOLT_WINRM_PASSWORD = $pass}
```

Comment:
I think the value of `$ENV:GITHUB_ENV` is a file that we should append our environment variable assignments to, not a variable that we should assign.